### PR TITLE
Use hooks to compile and clean Radius dictionaries

### PR DESCRIPTION
**Pull request by surik**
Previously we used rebar plugin to build Radius dictionaries but it only works with rebar2.
Using hooks allows us easy change to rebar3.

### CODE REVIEW

File: src/eradius_lib.erl [hunk 1]
```diff
@@ -469,7 +469,7 @@ salt_crypt(Op, SharedSecret, B, <<PlainText:16/binary, Remaining/binary>>, Ciphe
 	    end,
     salt_crypt(Op, SharedSecret, Bnext, Remaining, <<CipherText/binary, NewCipherText/binary>>);
 
-salt_crypt(Op, _SharedSecret, _B, << >>, CipherText) ->
+salt_crypt(_Op, _SharedSecret, _B, << >>, CipherText) ->
```
Comment (RoadRunnr) [inline src/eradius_lib.erl]: I don't mind that this warning fix goes into the same pull request, but could you please make it a separate commit?

Comment (surik) [inline src/eradius_lib.erl]: @RoadRunnr done

**Comment by RoadRunnr** [comment]
looks good
